marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/auth/LoginFailureHandler.java

```diff
@@ -37,7 +38,11 @@ public void onAuthenticationFailure(HttpServletRequest request, HttpServletRespo
         String json = node.toString();
         response.setContentType("application/json");
         response.setCharacterEncoding("UTF-8");
-        response.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
+        if (exception instanceof InsufficientAuthenticationException) {
```

Comment:
was this exception causing the clearUserData test to fail?

Reply:
I don't think so - that test was failing on 9.0-12 because it was depending on granular privileges that are in 10.0-3.

Reply:
I was referring to the unit test failure for the PR which I think runs on ML 10.0-4:
https://jenkins.marklogic.com/blue/organizations/jenkins/Datahub_CI/detail/PR-3914/16/tests

Reply:
It turned out that I was wiping out the granular privileges when attempting to deploy the amp/role for testing in ml-unit-tests here: https://github.com/marklogic/marklogic-data-hub/pull/3914/files#diff-1ec8825c599ca5cc7abf57c14482d7e5R79 

Adding the GranularPrivilegeCommand here addressed the problem.